ddtrace/tracer.py

```diff
@@ -331,6 +332,10 @@ def configure(
         if (collect_metrics is None and runtime_metrics_was_running) or collect_metrics:
             self._start_runtime_worker()
 
+        if self.log.level <= logging.INFO:
+            f = flare.jsonflare(self)
+            self.log.info("DD_TRACE_STARTUP_LOGS - %s", f)
+
```

Comment:
Can we print a line also for each diagnostics that failed? e.g. https://docs.google.com/document/d/146LTvy5WbfcnW5ESb-42sqRF7ACermfMKzvZVdfyJIM/edit?disco=AAAAGrrStA8

In your case I think you are only checking agent connectivity